src/D2L.CodeStyle.Analyzers/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 
 [assembly: ComVisible( false )]
 
-[assembly: AssemblyVersion( "0.9.0.0" )]
-[assembly: AssemblyFileVersion( "0.9.0.0" )]
+[assembly: AssemblyVersion( "0.9.1.0" )]

```

Comment:
I'm confused why you don't get a merge conflict here. `0.9.1.0` was taken recently: https://github.com/Brightspace/D2L.CodeStyle/blob/master/src/D2L.CodeStyle.Analyzers/Properties/AssemblyInfo.cs#L13